src/.nuget/nuget.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<config>
+		<add key="repositoryPath" value="../packages" />
+	</config>
+	<solution>
+		<add key="disableSourceControlIntegration" value="true" />
+	</solution>
+	<packageSources>
+		<add key="nuget.org" value="https://www.nuget.org/api/v2/" />
+	</packageSources>
+</configuration>
```

Comment:
Nuget can to have a global configuration. In my environment, i have some configuration like a repository local of the nuget and then i can't get external packages. With this local file Nuget configuration the solution he is armored.